include/souper/Extractor/Candidates.h

```diff
@@ -38,9 +38,11 @@ class Value;
 
 namespace souper {
 
+enum CandidateType { HarvestedFromUser, HarvestedFromUse };
```

Comment:
can you use an enum class here instead of an old-style enum?

Reply:
I think it may better reflect existing compiler terminology for these to be HarvestedFromDef, HarvestedFromUse

Reply:
Def is better since a User might not contain a return value for harvesting.